docs/adr/adr_backend_etl_and_hosting_for_StatViz.md

```diff
@@ -0,0 +1,237 @@
+# ADR: How to do etl in the backend for the DSEE100x project "StatViz"?
+
+Decision Deadline:
+
+Author: @pylipp, @HaGuesto
+
+## Status
+
+proposed
+
+## Context
+
+In the DSEE 100x project titled "StatViz", our goal is to construct data visualizations derived from Boxtribute's data. This will allow our users to showcase their contributions and impact to the public through various mediums, such as their websites or newsletters.
+
+Currently, data meant for these visualizations is planned to be served through a public GraphQL endpoint that is implemented using a stack of MySQL, Peewee, Flask, and Ariadne.
```

Comment:
Since performance is also of concern below, it might be worth explaining why we go for a GraphQL API here instead of REST GET endpoints (I think it might be less overhead to directly serve the data as JSON over REST rather having all the resolver overhead. OTOH we'll miss out on the implicit schema and type definitions)

Reply:
and What is your reasoning then?

I took the graphQL endpoint as a given :) just because we are accustomed to it.

Reply:
I'm also in favor of using GraphQL because it's better known to us, and because the schema is defined.